lib/travis/api/v3/remote_query.rb

```diff
@@ -56,7 +56,16 @@ def config
     end
 
     def s3_config
-      raise NotImplemented
+      conf = config.log_options.try(:s3) || {}
```

Comment:
I've done this. Everything works as expected. I've tested it on com-staging as well as org-staging.